### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by duttonw**

### CODE REVIEW

**Review by ThrawnCA — APPROVED** [review]
Some of these I committed myself, so someone else should review, but it looks good to me :)

File: pip-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+ckantoolkit==0.0.3
+pika>=1.1.0
+pyOpenSSL==18.0.0
+redis
+requests>=2.11.1
+six>=1.12.0
```
Comment (duttonw) [inline pip-requirements.txt:6]: do we need all of this for archiver? ie messytables?

File: requirements.txt [hunk 2]
```diff
@@ -4,5 +4,6 @@ progressbar==2.3
 #SQLAlchemy>=0.6.6
 #requests>=1.1.0
 ckantoolkit>=0.0.4
+six>=1.0.0
 
--e git+https://github.com/qld-gov-au/ckanext-report.git@0.1#egg=ckanext-report
\ No newline at end of file
+-e git+https://github.com/qld-gov-au/ckanext-report.git@0.1#egg=ckanext-report
```
Comment (duttonw) [inline requirements.txt]: are we still on 0.1?

File: pip-requirements.txt (hunk 1 above)
Comment (ThrawnCA) [reply pip-requirements.txt:6]: Probably not. Where did this file come from?

File: requirements.txt (hunk 2 above)
Comment (ThrawnCA) [reply requirements.txt]: 0.2, actually.

File: pip-requirements.txt (hunk 1 above)
Comment (duttonw) [reply pip-requirements.txt:6]: xloader setup

**Pull request merged**